src/CryptoAxus.Application/Features/Artist/GetArtistByWalletAddress/Handler/GetArtistDataByWalletAddress.cs

```diff
@@ -0,0 +1,33 @@
+﻿using CryptoAxus.Application.Features.Artist.GetArtistByWalletAddress.Request;
+using CryptoAxus.Application.Features.Artist.GetArtistByWalletAddress.Response;
```

Comment:
Using statements should be extracted to the relevant GlobalUsing file

Reply:
Move the using statements in the relevant GlobalUsing file

Reply:
Resolved